common/constants.go

```diff
@@ -219,3 +219,9 @@ const MemoKeyForOperator = "operator"
 
 // ReservedTaskListPrefix is the required naming prefix for any task list partition other than partition 0
 const ReservedTaskListPrefix = "/__cadence_sys/"
+
+// Enum for visibility message type
```

Comment:
Type is probably to generic name to understand. How about call it "visibilityOperation"? and we will have three operations: RecordStarted, RecordClosed and UpsertSearchAttributes